backend/src/main/java/codezap/member/configuration/BasicAuthentication.java

```diff
@@ -0,0 +1,4 @@
+package codezap.member.configuration;
+
+public @interface BasicAuthentication {
+}
```

Comment:
⚡
```suggestion
package codezap.member.configuration;

@Target(ElementType.PARAMETER)
@Retention(RetentionPolicy.RUNTIME)
public @interface BasicAuthentication {
}
```
어노테이션에 이 친구들 붙여줘야 동작하는 것 같은데 확인 부탁드립니다!!